/*
 * This file was automatically generated by EvoSuite
 * Tue Oct 04 14:46:47 KST 2016
 */


import static org.evosuite.runtime.EvoAssertions.assertThrownBy;
import org.junit.AfterClass;
import org.junit.BeforeClass;
import static org.junit.Assert.fail;

import org.evosuite.runtime.EvoRunner;
import org.evosuite.runtime.EvoRunnerParameters;
import org.junit.Test;
import org.junit.runner.RunWith;

@RunWith(EvoRunner.class) @EvoRunnerParameters(useVNET = true) 
public class ExplodierenEffekt_ESTest1   {
@BeforeClass
public static void initEvoSuiteFramework() {
}
@AfterClass
public static void exitEvoSuiteFramework() {
}

  @Test
  public void test00()  throws Throwable  {
      ExplodierenEffekt explodierenEffekt0 = new ExplodierenEffekt();
      explodierenEffekt0.x_Koe = (-1501);
      // Undeclared exception!
      try { 
        explodierenEffekt0.malen();
        fail("Expecting exception: NullPointerException");
      
      } catch(NullPointerException e) {
         //
         // no message in exception (getMessage() returned null)
         //
         assertThrownBy("ExplodierenEffekt", e);
      }
  }

  @Test
  public void test01()  throws Throwable  {
      ExplodierenEffekt explodierenEffekt0 = new ExplodierenEffekt();
      explodierenEffekt0.id = (-4776);
      // Undeclared exception!
      try { 
        explodierenEffekt0.malen();
        fail("Expecting exception: NullPointerException");
      
      } catch(NullPointerException e) {
         //
         // no message in exception (getMessage() returned null)
         //
         assertThrownBy("ExplodierenEffekt", e);
      }
  }

  @Test
  public void test02()  throws Throwable  {
      ExplodierenEffekt explodierenEffekt0 = new ExplodierenEffekt();
      explodierenEffekt0.colorG = (-2859);
      // Undeclared exception!
      try { 
        explodierenEffekt0.malen();
        fail("Expecting exception: NullPointerException");
      
      } catch(NullPointerException e) {
         //
         // no message in exception (getMessage() returned null)
         //
         assertThrownBy("ExplodierenEffekt", e);
      }
  }

  @Test
  public void test03()  throws Throwable  {
      ExplodierenEffekt explodierenEffekt0 = new ExplodierenEffekt();
      explodierenEffekt0.step = (-457);
      // Undeclared exception!
      try { 
        explodierenEffekt0.malen();
        fail("Expecting exception: NullPointerException");
      
      } catch(NullPointerException e) {
         //
         // no message in exception (getMessage() returned null)
         //
         assertThrownBy("ExplodierenEffekt", e);
      }
  }

  @Test
  public void test04()  throws Throwable  {
      ExplodierenEffekt explodierenEffekt0 = new ExplodierenEffekt();
      explodierenEffekt0.y_Koordinate = (-3900);
      // Undeclared exception!
      try { 
        explodierenEffekt0.malen();
        fail("Expecting exception: NullPointerException");
      
      } catch(NullPointerException e) {
         //
         // no message in exception (getMessage() returned null)
         //
         assertThrownBy("ExplodierenEffekt", e);
      }
  }

  @Test
  public void test05()  throws Throwable  {
      ExplodierenEffekt explodierenEffekt0 = new ExplodierenEffekt();
      explodierenEffekt0.objWidth_Koe = 1;
      // Undeclared exception!
      try { 
        explodierenEffekt0.malen();
        fail("Expecting exception: NullPointerException");
      
      } catch(NullPointerException e) {
         //
         // no message in exception (getMessage() returned null)
         //
         assertThrownBy("ExplodierenEffekt", e);
      }
  }

  @Test
  public void test06()  throws Throwable  {
      ExplodierenEffekt explodierenEffekt0 = new ExplodierenEffekt();
      explodierenEffekt0.objectType = (-1530);
      // Undeclared exception!
      try { 
        explodierenEffekt0.malen();
        fail("Expecting exception: NullPointerException");
      
      } catch(NullPointerException e) {
         //
         // no message in exception (getMessage() returned null)
         //
         assertThrownBy("ExplodierenEffekt", e);
      }
  }

  @Test
  public void test07()  throws Throwable  {
      ExplodierenEffekt explodierenEffekt0 = new ExplodierenEffekt();
      explodierenEffekt0.objHeight = 210;
      // Undeclared exception!
      try { 
        explodierenEffekt0.malen();
        fail("Expecting exception: NullPointerException");
      
      } catch(NullPointerException e) {
         //
         // no message in exception (getMessage() returned null)
         //
         assertThrownBy("ExplodierenEffekt", e);
      }
  }

  @Test
  public void test08()  throws Throwable  {
      ExplodierenEffekt explodierenEffekt0 = new ExplodierenEffekt();
      explodierenEffekt0.y_Koe = 4127;
      // Undeclared exception!
      try { 
        explodierenEffekt0.malen();
        fail("Expecting exception: NullPointerException");
      
      } catch(NullPointerException e) {
         //
         // no message in exception (getMessage() returned null)
         //
         assertThrownBy("ExplodierenEffekt", e);
      }
  }

  @Test
  public void test09()  throws Throwable  {
      ExplodierenEffekt explodierenEffekt0 = new ExplodierenEffekt();
      explodierenEffekt0.objHeight_Koe = (-2297);
      // Undeclared exception!
      try { 
        explodierenEffekt0.malen();
        fail("Expecting exception: NullPointerException");
      
      } catch(NullPointerException e) {
         //
         // no message in exception (getMessage() returned null)
         //
         assertThrownBy("ExplodierenEffekt", e);
      }
  }

  @Test
  public void test10()  throws Throwable  {
      ExplodierenEffekt explodierenEffekt0 = new ExplodierenEffekt();
      explodierenEffekt0.objWidth = 1769;
      // Undeclared exception!
      try { 
        explodierenEffekt0.malen();
        fail("Expecting exception: NullPointerException");
      
      } catch(NullPointerException e) {
         //
         // no message in exception (getMessage() returned null)
         //
         assertThrownBy("ExplodierenEffekt", e);
      }
  }

  @Test
  public void test11()  throws Throwable  {
      ExplodierenEffekt explodierenEffekt0 = new ExplodierenEffekt();
      explodierenEffekt0.x_Koordinate = (-1743);
      // Undeclared exception!
      try { 
        explodierenEffekt0.malen();
        fail("Expecting exception: NullPointerException");
      
      } catch(NullPointerException e) {
         //
         // no message in exception (getMessage() returned null)
         //
         assertThrownBy("ExplodierenEffekt", e);
      }
  }

  @Test
  public void test12()  throws Throwable  {
      ExplodierenEffekt explodierenEffekt0 = new ExplodierenEffekt();
      explodierenEffekt0.colorB = 1546;
      // Undeclared exception!
      try { 
        explodierenEffekt0.malen();
        fail("Expecting exception: NullPointerException");
      
      } catch(NullPointerException e) {
         //
         // no message in exception (getMessage() returned null)
         //
         assertThrownBy("ExplodierenEffekt", e);
      }
  }
}
